content/reference/sql/version-control/dolt-sql-procedures.md

```diff
@@ -419,6 +420,41 @@ CALL DOLT_COMMIT('-a', '-m', 'This is a commit', '--author', 'John Doe <johndoe@
 ```
 
 
+## `DOLT_CONFLICTS_RESOLVE()`
+
+When a merge finds conflicting changes, it documents them in the dolt_conflicts table.
+A conflict is between two versions: ours (the rows at the destination branch head) and theirs
+(the rows at the source branch head).
+`dolt conflicts resolve` will automatically resolve the conflicts by taking either
+the ours or theirs versions for each row.
+
+```sql
+CALL DOLT_CONFLICTS_RESOLVE('--ours', <table>);
+CALL DOLT_CONFLICTS_RESOLVE('--theirs', <table>);
+```
+
+### Options
+
+`<table>`: List of tables to be resolved. '.' can be used to resolve all tables.
+
+`--ours`: For all conflicts, take the version from our branch and resolve the conflict.
+
+`--theirs`: For all conflicts, take the version from their branch and resolve the conflict.
+
+### Examples
+
+```sql
+-- Set the current database for the session
+USE mydb;
+
+-- Attempt merge
+CALL DOLT_MERGE('feature-branch');
```

Comment:
Might have one additional step under this, where you do a select to see if there are conflicts